Sprint-1/interpret/percentage-change.js

```diff
@@ -11,12 +11,14 @@ console.log(`The percentage change is ${percentageChange}`);
 
 // Read the code and then answer the questions below
 
-// a) How many function calls are there in this file? Write down all the lines where a function call is made
+// a) How many function calls are there in this file? Write down all the lines where a function call is made (There are 3 function calls in this code, line 4 and 5, Number() is called twice)
 
-// b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem?
+// b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem? (I didn't get any error: The percentage change is 14.57)
```

Comment:
The statement at line 5 `priceAfterOneYear = Number(priceAfterOneYear.replaceAll("," ""));` should yields an error. 

There is a chance you fixed it before running the program.